ci/repipe

```diff
@@ -7,7 +7,7 @@ if [[ $(which ytt) == "" ]]; then
   exit 1
 fi
 
-target="${FLY_TARGET:-galoy}"
```

Comment:
why not just use "blink" here as the target?

Reply:
I don't like using the company name as some sort of fill-the-gap. It's IMHO not saying anything.

Reply:
ok, just keep it consistent across all pipelines so we don't need to look it up